### ISSUE TITLE
Installation bug with MySQL

### ISSUE THREAD

**Issue by ischilling** [zenphoto/zenphoto#930]
*Problem*
Clean install doesn't work always if configuration error with MySQL Server exists... the problem is that zenphoto installer does just white screen without any notification the moment the MySQL client data is entered and saved.

*Workaround*
If you change the zp-core/zenphoto_cfg.txt file and enter the MySQL Client data here, you get an error message that leads you to the problem ...

*Bug*
The installer doesn't show error messages but ends in a white screen

*System*
Most recent Debian Wheezy with all necessary updates/upgrades and most recent Apache (Debian rep.) + PHP (Debian rep.)

**Comment by acrylian** [comment]
Sorry, I never encountered this issue. 

>  you get an error message that leads you to the problem ...
And what error message would that be?

I would guess a permissions issue. You can create the config file manually, move it to `/zp-data` and rename it to `zenphoto.cfg.php`.

**Comment by ischilling** [comment]
:) well, that would have been helpful, that error message. 

 

In my case I did not correctly assign all Schema Privileges to the user – the error message was Access Denied.

 

As I wrote, when I did just change the configuration.txt and didn’t provide the correct file-system-rights, and added the user data, the setup did show the correct error message and worked as expected. If you do not propagate the correct login data in the configuration file and correct rights, it’s just a blank page after you entered all client data and pressed safe in the form.
